docs/config.rst

```diff
@@ -131,12 +131,12 @@ Optional settings
           config.processors << Raven::Processor::RemoveStacktrace
         end
 
-    By default, Sentry does not send POST data or cookies if present. To re-enable, remove the respective processor from the chain:
+    By default, Sentry does not send POST, PUT, PATCH data or cookies if present. To re-enable, remove the respective processor from the chain:
 
     .. code-block:: ruby
 
         Raven.configure do |config|
-          config.processors -= [Raven::Processor::PostData] # Do this to send POST data
+          config.sanitize_data_for_request_methods -= ["POST", "PUT", "PATCH"] # Do this to send POST, PUT PATCH data
```

Comment:
This is a little confusing. Can you make this example show how to only send "PATCH" data, and then add a (commented out) line below to show removing the processor entirely (and mention that this will send ALL request methods).